github-and-markdown/Markdown-Cheatsheet.md

```diff
@@ -0,0 +1,7 @@
+- [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+### _Further_:
+
+---
+
+...
```

Comment:
I think this is in the wrong place!

Reply:
But this is the markdown cheat sheet. I would keep it in the session for "github-and-markdown", no?